acceptance/tests/environment/negative/agent_run_should_fail_if_env_unreadable.rb

```diff
@@ -0,0 +1,58 @@
+test_name "C97899 - Agent run should fail if environment is unreadable" do
+  skip_test 'requires a master for managing the environment' if hosts_with_role(hosts, 'master').length == 0
+
+  testdir = ''
+  env_path = ''
+  test_env = ''
```

Comment:
i think `step` has its own scope, that's probably why they need instancing here.  then its value is used across two different steps